package.json

```diff
@@ -89,8 +89,7 @@
     "class-variance-authority": "^0.7.1",
     "clsx": "^2.1.1",
     "cmdk": "^1.1.1",
-    "convex": "^1.25.4",
-    "convex-oss-stats": "link:../../../erquhart/convex-oss-stats",
```

Comment:
I don't think this is going to work.

Reply:
This updates the convex package and removes a non-existent file based package, was left in accidentally from a while ago.